Dockerfile.tests

```diff
@@ -0,0 +1,11 @@
+FROM fedora:30
+
+RUN dnf install -y ansible
+
+COPY files/* /files/
```

Comment:
A name that is more descriptive than `files` would be welcome here

Reply:
I always use `files`.  It can be a `requirements` or `installations`. What are your preferences? I use `files` because it contains general stuff.

Reply:
how about `ansbile-requirements`? Or `image-requirements`, so it is obvious it is part of the image for testing

Reply:
`image-requirements` looks better. THX